phys2denoise/metrics/chest_belt.py

```diff
@@ -0,0 +1,272 @@
+    out_samplerate : :obj:`float`
+        Sampling rate for the output time series in seconds.
+        Corresponds to TR in fMRI data.
+    window : :obj:`int`, optional
+        Size of the sliding window, in the same units as out_samplerate.
+        Default is 6.
+    lags : (Y,) :obj:`tuple` of :obj:`int`, optional
+        List of lags to apply to the rv estimate. In the same units as
+        out_samplerate.
+
+    Returns
+    -------
+    env_arr
+
+    Notes
+    -----
+    This metric was first introduced in [1]_.
+
+    Across a sliding window, do the following:
+    1. Z-score respiratory belt signal
+    2. Calculate upper envelope
+    3. Calculate standard deviation of envelope
+
+    References
+    ----------
+    .. [1] J. D. Power et al., "Characteristics of respiratory measures in
+       young adults scanned at rest, including systematic changes and 'missed'
+       deep breaths," Neuroimage, vol. 204, 2020.
+    """
+    window = window * samplerate / out_samplerate
+    # Calculate RPV across a rolling window
+    env_arr = (
+        pd.Series(belt_ts).rolling(window=window, center=True).apply(rpv, window=window)
+    )
+    env_arr[np.isnan(env_arr)] = 0.0
+    return env_arr
+
+
+@due.dcite(references.CHANG_GLOVER_2009)
+def rv(belt_ts, samplerate, out_samplerate, window=6, lags=(0,)):
+    """Calculate respiratory variance.
+
+    Parameters
+    ----------
+    belt_ts : (X,) :obj:`numpy.ndarray`
+        A 1D array with the respiratory belt time series.
+    samplerate : :obj:`float`
+        Sampling rate for belt_ts, in Hertz.
+    out_samplerate : :obj:`float`
+        Sampling rate for the output time series in seconds.
+        Corresponds to TR in fMRI data.
+    window : :obj:`int`, optional
+        Size of the sliding window, in the same units as out_samplerate.
+        Default is 6.
+    lags : (Y,) :obj:`tuple` of :obj:`int`, optional
+        List of lags to apply to the rv estimate. In the same units as
+        out_samplerate.
+
+    Returns
+    -------
+    rv_out : (Z, 2Y) :obj:`numpy.ndarray`
+        Respiratory variance values, with lags applied, downsampled to
+        out_samplerate, convolved with an RRF, and detrended/normalized.
+        The first Y columns are not convolved with the RRF, while the second Y
+        columns are.
+
+    Notes
+    -----
+    Respiratory variance (RV) was introduced in [1]_, and consists of the
+    standard deviation of the respiratory trace within a 6-second window.
+
+    This metric is often lagged back and/or forward in time and convolved with
+    a respiratory response function before being included in a GLM.
+    Regressors also often have mean and linear trends removed and are
+    standardized prior to regressions.
+
+    References
+    ----------
+    .. [1] C. Chang & G. H. Glover, "Relationship between respiration,
+       end-tidal CO2, and BOLD signals in resting-state fMRI," Neuroimage,
+       issue 4, vol. 47, pp. 1381-1393, 2009.
```

Comment:
A bit of a nit-pick, but I *think* RV was described in "Influence of heart rate on the BOLD signal: the cardiac response function" also by Chang and Glover in 2009. It may also be in this paper, but I didn't see it from a skim.

Reply:
I think @smoia noticed similar issues with my citations. We can address in a new PR, or you can fold it into #21?

Reply:
I think we should handle it in a new PR